kythe/java/com/google/devtools/kythe/extractors/java/standalone/javac-wrapper.sh

```diff
@@ -71,8 +68,8 @@ fi
 
 "$JAVABIN" "${KYTHE_JAVA_RUNTIME_OPTIONS[@]}" \
   -jar "$JAVAC_EXTRACTOR_JAR" "$@" \
-  1>>"$TMPDIR"/javac-extractor.out \
-  2>>"$TMPDIR"/javac-extractor.err
+  1> >(sed -e 's/^/EXTRACT OUT: /') \
+  2> >(sed -e 's/^/EXTRACT ERR: /' >&2)
```

Comment:
Oh, GCB only gives you stderr? Where does stdout go?

Reply:
No GCB does give you stdout too

My bash is not super strong, but my understanding is that this will just prepend 'EXTRACT OUT' in front of stdout and 'EXTRACT ERR' in front of stderr

Reply:
Crucially, for certain classes of errors (like the java11 one), the useful message is printed to stdout.  I went and tested that this message shows up:

```
Step #5 - "EXTRACT": EXTRACT OUT: -Xbootclasspath/p is no longer a supported option.
```

Reply:
Ah, didn't see why we'd want to distinguish them. Nothing else does; I'm not sure if it's the right/best thing to do to be "different" but this is fine.

Reply:
Mostly I just wanted to distinguish these from other debug printing, so I started with EXTRACT ERR as a prefix in there.

Then I figured prefixing stdout messages with EXTRACT ERR felt a bit odd, so I changed that to EXTRACT OUT.

But we could easily change this later.